integration-tests/test-programs/spawn-child.c

```diff
@@ -0,0 +1,22 @@
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h>
+
+int main() {
+    usleep( 100000 );
+    malloc( 10001 );
+
+    pid_t pid = fork();
+    if( pid == 0 ) {
+        // Child
+        if (execl("/usr/bin/ls", "/usr/bin/ls", NULL) == -1) {
```

Comment:
Could you use the `basic.c` program here instead and in the integration test verify that:
1) the file for the parent is created and contains the allocations we expect,
2) the file for the child is created and contains the allocations we expect (you can just grab the checks from `#[test]` for the `basic.c`, move it into a separate functions, and then call it in two places)

Reply:
Good idea, I changed the test to incorporate the `basic` program.